scripts/jenkins-test

```diff
@@ -37,18 +37,59 @@ then
     exit 1
 fi
 
+# this next line is supposed to fail
+set +e
+
+echo "Rewriting POM.xml files to Scala 2.10 and Spark 1 should error..."
+./scripts/move_to_spark_1.sh
+if [[ $? == 0 ]];
+then
+    echo "Moving to Spark 1 did not cowardly fail!"
```

Comment:
"did not cowardly fail" is not as clear as, for example, "completed updating pom.xml for spark1"